### DISCUSSION TITLE
disable access.log

### DISCUSSION THREAD

**Discussion by Tealk** [NginxProxyManager/nginx-proxy-manager#2131] [Q&A]
Hi,

is it possible to disable the access.log's

**Comment by mrrfv** [comment]
Haven't tested this, but it looks like it's doable. Simply add this to your proxy/redirect/other host's advanced configuration (or as a snippet) to disable both the access and error logs:

```
location / {
    access_log /dev/null;
    error_log /dev/null crit;
}
```
